PanModal/View/DimmedView.swift

```diff
@@ -57,11 +57,11 @@ public class DimmedView: UIView {
 
     // MARK: - Initializers
 
-    init(dimAlpha: CGFloat = 0.7) {
+    init(dimColor: UIColor, dimAlpha: CGFloat) {
```

Comment:
In this case, we can probably remove `dimAlpha` completely as this is already a property on `UIColor` - this avoids conflicting dim logic e.g. passing `.black with 0.2 alpha` dimColor but dimAlpha is `0.7`